lib/dest/index.js

```diff
@@ -12,11 +12,21 @@ function dest(outFolder, opt) {
   }
 
   function saveFile(file, enc, cb) {
+    /* jshint validthis: true */
+    var stream = this;
+
+    function complete(err, file) {
+      cb(err, file);
+      if (!err) {
```

Comment:
> ; however, that has no bearing on if we can read more

Ahh i guess I misunderstood
>  _read should start pushing that data into the read queue by calling this.push(dataChunk). _read should continue reading from the resource and pushing data until push returns false, at which point it should stop reading from the resource...

https://nodejs.org/api/stream.html#stream_readable_read_size_1


> so the read will essentially nothing in that case (I think).

yeah looks like it returns null when `pipe` from `dest`